embc-app/ClientApp/src/app/volunteer-list/volunteer-list.component.html

```diff
@@ -4,24 +4,21 @@ <h1>Emergency Support Services Team Management Dashboard</h1>
   <p>Use this section to add new Emergency Support Services users, as well as edit existing user information and permissions.</p>
 
   <h3 class="blue-header">USERS LIST</h3>
-  <form novalidate role="search" [formGroup]="form">
-    <div class="form-row">
-      <div class="col-md">
-        <input type="text" aria-label="search" class="form-control mb-2" placeholder="Search by last name &hellip;" formControlName="searchbox" (keydown.enter)="search()">
-      </div>
-      <div class="col-md-auto">
-        <button class="btn btn-primary" (click)="search()">SEARCH</button>
-      </div>
+  <div class="form-row">
+    <div class="col-md">
+      <input type="text" aria-label="search" class="form-control mb-2" placeholder="Search by last name &hellip;" [(ngModel)]="queryString" (keydown.enter)="search()">
+    </div>
+    <div class="col-md-auto">
+      <button class="btn btn-primary" (click)="search()">SEARCH</button>
     </div>
-  </form>
+  </div>
   <div class="search-results mt-4">
-    <div *ngIf="results && results.length > 0; then searchResultsTable; else notFound"></div>
+    <div *ngIf="resultsAndPagination && resultsAndPagination.data.length > 0; then searchResultsTable; else notFound"></div>
```

Comment:
FYI, first part could be simplified to...
*ngIf="resultsAndPagination?.data?.length > 0; then ...